src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,99 @@
+package edu.kis.vh.nursery;
+
+import edu.kis.vh.nursery.list.IntArrayStack;
+import edu.kis.vh.nursery.list.IntLinkedList;
+import edu.kis.vh.nursery.list.ListInterface;
+
+public class DefaultCountingOutRhymer 
+// TODO what this does actually?
+{
+	public void push(int in) {
+		NUMBERS.push(in);
+	}
+
+
+	public boolean isEmpty() {
+		return NUMBERS.isEmpty();
+	}
+
+
+	public int top() {
+		return NUMBERS.top();
+	}
+
+
+	public int pop() {
+		return NUMBERS.pop();
+	}
+
+
+	private static final int MINUSE_ONE = 0;
```

Comment:
It doesn't make sense to me. Does it make sense for you? Actually this constant should finally go to the ListInterface. Extracting constants is done incorrectly. Have you asked for review hints (1.4.w7): tiny.cc/998muz ?
This constant should be different (separate) than the one in peekaboo and countOut methods.